go/sqltypes/type.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2015, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
The comment is in value.go. Some go packages follow the standard of creating a separate doc.go file for each package to avoid this confusion. We haven't talked about switching to that standard yet.